repos/core-setup.proj

```diff
@@ -9,25 +9,30 @@
     <OverridePortable>$(PortableBuild)</OverridePortable>
     <OverridePortable Condition="'$(TargetOS)' == 'OSX'">true</OverridePortable>
 
-    <BuildArguments>-ConfigurationGroup=$(Configuration)</BuildArguments>
-    <BuildArguments>$(BuildArguments) -PortableBuild=$(OverridePortable)</BuildArguments>
-    <BuildArguments>$(BuildArguments) -SkipTests=true</BuildArguments>
+    <BuildArguments>--restore --build</BuildArguments>
+    <BuildArguments>$(BuildArguments) --configuration $(Configuration)</BuildArguments>
+    <BuildArguments>$(BuildArguments) --ci</BuildArguments>
+    <BuildArguments>$(BuildArguments) /p:PortableBuild=$(OverridePortable)</BuildArguments>
     <BuildArguments Condition="$(Platform.Contains('arm'))">$(BuildArguments) -TargetArchitecture=$(Platform) -DisableCrossgen=true -CrossBuild=true</BuildArguments>
-    <BuildArguments>$(BuildArguments) --</BuildArguments>
     <BuildArguments>$(BuildArguments) /p:BuildDebPackage=false</BuildArguments>
     <BuildArguments>$(BuildArguments) /p:BuildAllPackages=true</BuildArguments>
+    <BuildArguments>$(BuildArguments) /p:RestoreAllBuildRids=false</BuildArguments>
```

Comment:
Filed https://github.com/dotnet/core-setup/issues/7629 to track being able to remove this.

Reply:
Thanks!